decision-process/README.md

```diff
@@ -0,0 +1,134 @@
+---
+creation_date: 2023-10-10
+issues:
+  - https://github.com/giantswarm/giantswarm/issues/24661
+owners:
+  - https://github.com/orgs/giantswarm/teams/sig-architecture
```

Comment:
Why did you go with URL over just the team name?

Reply:
Same with the `docs` repo (where the field is called `owner` – I changed that to plural _here_). It's also easier to verify whether it points to an actual GitHub group, as we desire here for future automation.

Reply:
works for me, sure